src/extensions/idle.rs

```diff
@@ -31,6 +31,15 @@ pub struct Handle<'a, T: Read + Write> {
     done: bool,
 }
 
+/// The result of a wait on a `Handle`
```

Comment:
```suggestion
/// The result of a wait on a [`Handle`]
```
This makes it a clickable link!